### PR TITLE
Component changelog

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
Added changelog for is_component_loaded function optimization.

### Types of changes
- Improvements.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -14,6 +14,7 @@ v3.0.0
 - Improvement: Builder - Menu - Added menu hover animation styles.
 - Improvement: Builder - Off-Canvas - Added container alignment option.
 - Improvement: Removed extra localized variable declaration.
+- Improvement: Optimize is_component_loaded function to reduce unwantend dynainc CSS loading.
```
Comment (vrundakansara) [inline changelog.txt:17]: This needs to be added under v3.1.0 and not under v3.0.0

Comment (sagarb90) [reply changelog.txt]: @vrundakansara This is Done.

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,6 @@
+v3.1.0
+- Improvement: Optimize is_component_loaded function to reduce unwantend dynainc CSS loading.
```
Comment (vrundakansara) [inline changelog.txt:2]: `- Improvement: Optimized dynamic CSS functionality to reduce unwanted CSS on the page.`
Fix the typos and rename it to this.

Comment (sagarb90) [reply changelog.txt]: @vrundakansara Done!

**Review by vrundakansara — APPROVED** [review]

**Review by rohitp-bsf — APPROVED** [review]

**Pull request merged**